tests/issues.targets

```diff
@@ -891,6 +894,12 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Tests that run only on x86 Windows -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and ('$(BuildArch)' != 'x86' or '$(TargetsWindows)' != 'true')">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/more_tailcalls/*">
+            <Issue>x64 Windows has bugs with helper-based tailcall to VSD; rest of platforms do not support it</Issue>
```

Comment:
Is there an open issue tracking this?

Reply:
If not, can you open one and add the issue number here?

Reply:
Will do, this issue is #26311. I believe it is the cause of a number of other issues opened recently, like #25960 and #26673.